### DISCUSSION TITLE
Cannot download latest EP

### DISCUSSION THREAD

**Discussion by TomasL-SE** [valinet/ExplorerPatcher#3618] [General]
Latest build 23H2 22632.4112.
EP_Setup is classiifed as a virus, and windows refuses to donload it regardless what I do.

**Comment by pyrates999** [comment]
Duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3228

Please close this.

MS just doesn't like EP and has added it to windows defender, even though it is not a virus and not a hack tool.

Also set windows defender to exclude the following:

1. %ProgramFiles%\ExplorerPatcher
2. %APPDATA%\ExplorerPatcher
3. Directory that you downloaded ep_setup.exe to

Future updates to EP won't be flagged then.

**Comment by regs01** [comment]
It's Microsoft and mostly low quality solutions, which are likely paid by Microsoft, that threat it as a malware, although hack tools are not malware. But this isn't even a hack tool. Aside that there is only Alibaba/AliCloud, but they just copying Microsoft.

Out of all other independent and well known anti-virtus solutions, like Avira, ClamAV, DrWeb, NOD32, Kaspersky, NANO, Panda, Symantec, Tencent, TrendMicro etc see it clean.
https://www.virustotal.com/gui/file/a1a736c78531e03325a2beb70a4d802ba81adaae63f530c00fb8f8c96ce375ae/detection/f-a1a736c78531e03325a2beb70a4d802ba81adaae63f530c00fb8f8c96ce375ae-1725040785
